html/inc/boinc_db.inc

```diff
@@ -505,6 +545,11 @@ class BoincProfile {
         $db = BoincDb::get();
         return $db->insert('profile', $clause);
     }
+    /**
+     * @param string|null $where_clause
+     * @param string|null $order_clause
+     * @return list<BoincProfile>
+     */
     static function enum($where_clause=null, $order_clause=null) {
```

Comment:
This is the only occurence of `$where_clause=null`, should I keep it? Or should I check for usages?